### PR TITLE
Activate versioning stages in pipeline and update generated files

### PR DESCRIPTION
**Pull request by theunrepentantgeek**
**What this PR does / why we need it**:

Enables the disabled stages in the pipeline to generate the `Hub()` and related conversion methods.

**Prerequisites**

- [x] Merge PR #1935
- [x] Merge PR #1985 
- [x] Merge PR #1973

**How does this PR make you feel**:
gif

### CODE REVIEW

**Review by babbageclunk — APPROVED** [review]
Looks great - thanks for splitting out the generated changes into separate commits like that, made it heaps easier to review!

**Review by babbageclunk — APPROVED** [review]
I've approved this before, but the latest version looks good as well.

File: v2/api/authorization/v1alpha1api20200801preview/role_assignment_types_gen.go [hunk 1]
```diff
@@ -14,15 +14,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
-// +kubebuilder:rbac:groups=authorization.azure.com,resources=roleassignments,verbs=get;list;watch;create;update;patch;delete
```
Comment (matthchr) [inline v2/api/authorization/v1alpha1api20200801preview/role_assignment_types_gen.go:20]: Did we lose these comments? I think we need them?

Comment (matthchr) [reply v2/api/authorization/v1alpha1api20200801preview/role_assignment_types_gen.go:20]: Oh, they just got moved into the storage version file... that's _probably_ ok.

**Review by matthchr — APPROVED** [review]

**Pull request merged**